.github/workflows/TestAcquia.yml

```diff
@@ -70,10 +90,10 @@ jobs:
           echo "/drainpipe" >> .drainpipeignore
           ddev task snapshot:directory directory=/tmp/release
 
-      - name: Clone from production to dev
+      - name: Clone from test to dev
         uses: ./drainpipe/scaffold/github/actions/acquia/clone-env
         with:
-          source-environment: lullabotsandbox.prod
+          source-environment: lullabotsandbox.test
```

Comment:
@beto-aveiga were we able to get this working w/ prod after we notified we were having issues testing with prod in our Sandbox?

Reply:
Sure! I can try it now.
I think there will be no issues.